resume/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Section from "./components/Section";
+import resumeJSON from "./data/resume";
+import Basics from "./components/sections/Basics";
+import Work from "./components/sections/Work";
+
+import "./App.css";
+
+function App() {
+  let counter = 0;
+  const sections = Object.keys(resumeJSON).map(section => {
+    counter++;
```

Comment:
the callback in `.map` gives you counts as the 2nd parameter, right after `section`.

```
.map((section, counter) => { 
//..
}
```